call-hierarchy-sample/src/model.ts

```diff
@@ -0,0 +1,91 @@
+import {
+	Range, Position, CallHierarchyProvider, TextDocument, CancellationToken, CallHierarchyItem,
+	SymbolKind, ProviderResult, CallHierarchyIncomingCall, CallHierarchyOutgoingCall, workspace, Uri
+} from 'vscode';
+
```

Comment:
suggestion to use `import * as vscode from ...` like its done in the extension file

Reply:
It makes the code more verbose, but perhaps more readable. And uniform, so modified. Done. Thanks